contrib/linux/requirements.txt

```diff
@@ -1,3 +1,4 @@
 # used by file watcher sensor
 pyinotify>=0.9.5,<=0.10
 -e git+https://github.com/Kami/logshipper.git@stackstorm_patched#egg=logshipper
+ipaddress>=1.0.16
```

Comment:
(Just thinking out loud) - Module license is Python Software Foundation License v2 (https://github.com/phihag/ipaddress/blob/master/LICENSE) so it's OK for us to bundle this dependency.